data/package_index.yml

```
@@ -1626,3 +1626,11 @@
   tags: quasi-newton-optimization stochastic-optimization acm-toms
   license: MIT
   version: none
+
+- name: FLAME
+  gitlab: flame-code/FLAME
+  description: FLAME: a library for atomistic modeling environments.
+  categories: atomistic simulation
```

```suggestion
  categories: scientific
```

Valid categories are given here: https://fortran-lang.org/community/packages/#github-hosted-packages

---

Hi Ivan

  I added GitLab Pages to the README and also url to repo on GitLab is added to the documentation in section "Download" right after the link to the repo on GitHub.